### PR TITLE
Dev logger

### PR DESCRIPTION
**Pull request by vongang**
screen shot 2017-06-05 at 10 22 35 pm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ install:
   - go get github.com/spf13/viper
   - go get github.com/bitly/go-simplejson
   - go get github.com/stretchr/testify/assert
+  - go get github.com/apsdehal/go-logger
```
Comment (vaporz) [inline .travis.yml]: this package seems more powerful:
github.com/sirupsen/logrus

**Comment by vaporz** [comment]
And, how to set log levels?

File: generator.go [hunk 2]
```diff
@@ -497,7 +500,7 @@ import (
 )
 
 func RegisterServer(s *grpc.Server) {
-	proto.RegisterTestServiceServer(s, &{{.ServiceName}}{})
+	proto.Register{{.ServiceName}}Server(s, &{{.ServiceName}}{})
```
Comment (vaporz) [inline generator.go]: Nice! Thank you!

**Comment by vongang** [comment]
it's a simple logger, and there is 6 log level: Fatal, Error, Warning, Notice, Info, and Debug.
just use log.Fatal(message) to log the fatal level message.

File: .travis.yml (hunk 1 above)
Comment (vongang) [inline .travis.yml]: I will investigate it

**Comment by vongang** [comment]
log lib has changed to logrus.

**Comment by vaporz** [comment]
CI failed

File: generator.go [hunk 3]
```diff
@@ -36,14 +37,14 @@ func validateServiceRootPath() {
 	if os.IsNotExist(err) {
 		return
 	}
-	fmt.Print("Path '" + Config.ServiceRootPath() + " already exist!\n" +
+	log.Warn("Path '" + Config.ServiceRootPath() + " already exist!\n" +
```
Comment (vaporz) [inline generator.go]: my mistake, command line tools should output to stdout, use "fmt", not log

**Pull request closed without merging**